omnibus/config/software/chefdk.rb

```diff
@@ -39,17 +39,13 @@
 
 relative_path "chef-dk"
 
-if windows?
-  dependency "ruby-windows"
-  dependency "ruby-windows-devkit"
-else
-  dependency "libffi" if debian?
-  dependency "ruby"
-end
+dependency "ruby"
 
 dependency "rubygems"
 dependency "bundler"
 dependency "appbundler"
+# windows does not have native readline support with compiled ruby
+dependency "rb-readline" if windows?
```

Comment:
I actually did this due to the fact that we don't compile with readline at all--this is necessary for the software def to work at present.